docs/Catalog/VisualizationDashboards.md

```diff
@@ -350,3 +352,22 @@ All filters and columns will be restored:
   }
 ]
 ```
+
+## Igv
+
+To render an [Igv](https://igv.org/), you provide a JSON file (a dictionary that
+specifies the Igv [browser configuration options](https://github.com/igvteam/igv.js/wiki/Browser-Creation#browser-configuration-options))
+and you set the `"types"` property to `[ "igv" ]`.
+
+Alternatively, you can browse that JSON file in Quilt web and switch viewer from JSON to Igv
+
+```json
+// quilt_summarize.json
+[
+  {
+    "path": "igv-options-file.json",
+    "title": "Awesome genome",
+    "types": ["igv"]
+  }
+]
```

Comment:
```suggestion

## Integrative Genomics Viewer (IGV)


```suggestion
To render genome tracks, you can select "View as IGV" in the catalog,
or you can invoke [igv.js](https://igv.org/) with quilt_summarize, as shown below:
```json
// quilt_summarize.json
[
  {
    "path": "igv-options-file.json",
    "title": "Awesome genome",
    "types": ["igv"]
  }
]
```
In the above example, `igv-options-file.json` is an
[IGV browser configuration](https://github.com/igvteam/igv.js/wiki/Browser-Creation#browser-configuration-options).
```